docs/sphinx/source/user_guide/clearsky.rst

```diff
@@ -642,7 +642,8 @@ Now we run the synthetic data and clear sky estimate through the
 
 .. ipython:: python
 
-    clear_samples = clearsky.detect_clearsky(ghi, cs['ghi'], cs.index, 10)
+    clear_samples = clearsky.detect_clearsky(ghi, cs['ghi'], cs.index,
+                     window_length=10)
```

Comment:
```suggestion
    clear_samples = clearsky.detect_clearsky(ghi, cs['ghi'])
```

On second thought, I think we should just get rid of these optional inputs.  They used to be required, but we've since updated the function so that they don't have to be specified explicitly anymore.  Better to keep the examples simple and on one line when we can.